src/input.ts

```diff
@@ -41,3 +41,13 @@ export function getInputList(
         .map((item: string) => item.trim())
         .filter((item: string) => item.length > 0);
 }
+
+export function getInputAsArray(
```

Comment:
This function is used in `actions-rs/install` and copied from `@actions/core`, but using the `getInput` wrapper function of this package.